### DISCUSSION TITLE
RenderTexture and Transparency

### DISCUSSION THREAD

**Discussion by gabrielliwerant** [pixijs/pixijs#10040] [General]
Is there any way to make RenderTexture respect transparency in sprites? I have sprites with transparent areas and they show up as a black when I create a RenderTexture with them.

**Comment by bigtimebuddy** [comment]
Here's an example of how to use a RenderTexture:
https://pixijs.com/examples/textures/render-texture-basic

Do you have reproduction? Are you using compressed texture formats?

**Reply by gabrielliwerant** [reply]
I was using PNGs with transparency, but it turns out that my issue was with the screen dimensions, which led to some of the sprites being cut off, not a RenderTexture issue. Thank you for your help!
